nilearn/surface/surface.py

```diff
@@ -830,7 +832,8 @@ def load_surf_mesh(surf_mesh):
 
     Parameters
     ----------
-    surf_mesh : str, pathlib.Path, or numpy.ndarray or Mesh
+    surf_mesh : :obj:`str`, :obj:`pathlib.Path`, or :obj:`numpy.ndarray`
+        or Mesh
```

Comment:
multi-line type definitions need a `\` at the end of where the line breaks:

Otherwise, it renders like this: 
![Screenshot from 2023-07-21 11-37-48](https://github.com/nilearn/nilearn/assets/63292494/a9c08b41-1dd6-415b-a20d-7650a50c0936)


Example: https://github.com/nilearn/nilearn/blob/13fad00c95adbffdcdfbcde3f1062cb6391333e9/nilearn/glm/first_level/first_level.py#L437-L438